Case file for DIVD-2023-00016 (#594)
DIVD-NL/web-csirt: _cases/2023/DIVD-2023-00016.md

---
layout: case
title: "GLPI Remote Code Execution"
author: Finn van der Knaap en Josha Beekman
lead: Josha Beekman
status: Open
excerpt: "GLPI version below 9.5.9 & 10.0.3 are vulnerable to Remote Code Execution"
researchers:
- Finn van der Knaap
- Josha Beekman
cves:
- CVE-2022-35914
product: GLPI

> versions: < 9.5.9 & 10.0.3

Reviewer: ```suggestion
versions: 
- “< 9.5.9”
- “< 10.0.3”
```

> recommendation: "Update to the latest version or delete the vendor/htmlawed/htmlawed/htmLawedTest.php file (be careful not to touch the htmLawed.php file which is legitimate)."

Reviewer: Is dat laatste gee workaround?
```suggestion
recommendation: "Update to the latest version”
workaround: “Delete the vendor/htmlawed/htmlawed/htmLawedTest.php file (be careful not to touch the htmLawed.php file which is legitimate)."
```

start: 2022-11-10
end: 
timeline:
- start: 2022-12-10
  end:
  event: "Started research"
- start: 2022-12-16
  end: 2022-12-17
  event: "Nuclei template made"
- start: 2022-12-19
  end: 2023-03-04
  event: "Started scanning"
  
---

## Summary

> The case involves a code injection vulnerability in GLPI instances versions <=10.0.2 and <=9.5.8, caused by an old version of the htmlawed library (under /vendor/htmlawed/htmlawed/) that still contains the htmLawedTest.php file with code injection vulnerability.

Reviewer: <10.0.3 is hetzelfde als <= 10.0.2, maar probeer het zoveel mogelijk hetzelfde te doen.

> ## What you can do

Reviewer 2: Misschien nog even consistent newlines boven/onder paragrafen plaatsen

---
layout: case
title: "GLPI Remote Code Execution"
author: Finn van der Knaap en Josha Beekman
lead: Josha Beekman
status: Open
excerpt: "GLPI version below 9.5.9 & 10.0.3 are vulnerable to Remote Code Execution"
researchers:
- Finn van der Knaap
- Josha Beekman
cves:
- CVE-2022-35914
product: GLPI
versions: 
- “< 9.5.9”
- “< 10.0.3”

> recommendation: "Update to the latest version”
> workaround: “Delete the vendor/htmlawed/htmlawed/htmLawedTest.php file (be careful not to touch the htmLawed.php file which is legitimate)."

Reviewer: ```suggestion
recommendation: "Update to the latest version"
workaround: "Delete the vendor/htmlawed/htmlawed/htmLawedTest.php file (be careful not to touch the htmLawed.php file which is legitimate)."
```
Stomme smart quotes op mij telefoon.

start: 2022-11-10
end: 
timeline:
- start: 2022-12-10
  end:
  event: "Started research"
- start: 2022-12-16
  end: 2022-12-17
  event: "Nuclei template made"
- start: 2022-12-19
  end: 2023-03-04
  event: "Started scanning"
  
---

## Summary

> The case involves a code injection vulnerability in GLPI instances versions <=10.0.2 and <=9.5.8, caused by an old version of the htmlawed library (under /vendor/htmlawed/htmlawed/) that still contains the htmLawedTest.php file with code injection vulnerability.

Reviewer: ```suggestion
The case involves a code injection vulnerability in GLPI instances versions < 10.0.3 and < 9.5.9, caused by an old version of the htmlawed library (under /vendor/htmlawed/htmlawed/) that still contains the htmLawedTest.php file with code injection vulnerability.
```
